src/components/Modal/Modal.tsx

```diff
@@ -145,11 +143,15 @@ export function ModalHeader({ children }: { children: ReactNode }): JSX.Element
 }
 
 /** Provides consistent styling and the scrolling behavior for a modal's primary content. */
-export function ModalBody({ children }: { children: ReactNode }): JSX.Element {
+export function ModalBody({
+  children,
+  virtualized = false,
```

Comment:
Thought: It's probably too cute, but iirc this is our 2nd "just know to set `virtualized` based on what you're putting in a parent" ... might be nice if there was something that would let a child tell it's parent "hey i'm virtualized".

Like in gridtable:

```
function GridTable(props) {
  const { setVirtualized } = useBeamContext();

   setVirtualized(props.virtualized);
}
```

And then "special" containers like `ScrollableParent` (?) and `ModalBody` would have providers listening for "ah yeah, my child is actually virtualized" and use that to drive their `virtualized` state/behavior instead of an explicit prop.

@bdow wdyt?